packages/editor/src/components/url-popover/README.md

```diff
@@ -0,0 +1,142 @@
+URLPopover
+===========
+
+URLPopover is a presentational React component used to render a popover used for editing and viewing a url.
+
+## Setup
+
+The component will be rendered adjacent to its parent.
+
+```jsx
+import { ToggleControl, IconButton } from '@wordpress/components';
+import { URLPopover } from '@wordpress/editor';
+
+class MyURLPopover extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.onChangeURL = this.onChangeURL.bind( this );
+		this.openURLPopover = this.closeURLPopover.bind( this );
+		this.closeURLPopover = this.closeURLPopover.bind( this );
+		this.submitURL = this.submitURL.bind( this );
+		this.editURL = this.editURL.bind( this );
+		this.setTarget = this.setTarget.bind( this );
+
+		this.state = {
+			isEditing: true,
+			isVisible: false,
+		};
+	}
+
+	onChangeURL( url ) {
+		this.setState( { url } );
+	}
+
+	openURLPopover() {
```

Comment:
I'm not seeing anything in this example code that calls `openURLPopover`. Should we add a `<button>` that demonstrates it? Or just remove it?

Reply:
Done